src/oss/langchain-mcp.mdx

```diff
@@ -356,17 +384,87 @@ app.listen(PORT, () => {
 :::
 
 :::python
-## Additional resources
+## Expose LangChain tools via MCP
 
-* [MCP documentation](https://modelcontextprotocol.io/introduction)
-* [MCP Transport documentation](https://modelcontextprotocol.io/docs/concepts/transports)
-* [langchain_mcp_adapters](https://github.com/langchain-ai/langchain-mcp-adapters)
+You can also expose existing LangChain tools through an MCP server using the `to_fastmcp` function. This allows you to make your LangChain tools available to any MCP client.
+
+```python
+from langchain_core.tools import tool
+from langchain_mcp_adapters.tools import to_fastmcp
+from mcp.server.fastmcp import FastMCP
+
+
+@tool
+def add(a: int, b: int) -> int:
+    """Add two numbers"""
+    return a + b
+
+@tool
+def get_user_info(user_id: str) -> str:
+    """Get information about a user"""
+    return f"User {user_id} is active"
+
+
+# Convert LangChain tools to FastMCP
+fastmcp_tools = [to_fastmcp(add), to_fastmcp(get_user_info)]
+
+# Create MCP server with the converted tools
+mcp = FastMCP("LangChain Tools", tools=fastmcp_tools)
+mcp.run(transport="stdio")
+```
+
+## Stateless architecture
+
+The MCP client is **stateless by default**, which provides several important benefits:
+
+- **Scalability**: Web servers can handle requests without maintaining session state, making horizontal scaling simpler
```

Comment:
Let's remove the benefits -- it's too much text and this is just the client not the protocol or the server.

---

* Could be be specific and explain that `MultiServerMCPClient` is stateless.
* If folks need to use MCP in a statefull manner, they can manage the session objects on their own